app/src/main/java/com/twilio/video/app/ui/room/RoomViewModel.kt

```diff
@@ -314,16 +317,18 @@ class RoomViewModel(
         }
     }
 
+    @ExperimentalCoroutinesApi
```

Comment:
Why is this annotation required for launching a co-routine?

Reply:
Oh it's only needed for [line 152](https://github.com/twilio/twilio-video-app-android/pull/153/files#diff-0d9d241051d0102101681029e79737df2dd270f26508aec54219a87464612bb5R152) when calling the `recieveOrNull` function. This a convenience function that will either suspend or return null which makes it easy to subscribe to new items emitted to the channel and handling the case when the channel is cancelled. If that API changes in a future kotlin release it will be an easy change.

Reply:
I tried to avoid use of this annotation as much as possible. I wanted to use a `StateFlow` instead of a `Channel`, but it is still experimental. As soon as it's stable, I think we should move to it as it is easier to work with and has auto cancel capabilities